src/api/app/controllers/webui/package_controller.rb

```diff
@@ -701,9 +701,10 @@ def save_file
           errors << 'No file or URI given'
         end
       end
-
-    rescue APIException, StandardError => exception
-      errors << exception.message
+    rescue ActiveXML::Transport::Error => e
```

Comment:
Hm this potentially can mean many things. 40x, network unreachable etc. You are aware of this?